shell/client/shell-client.js

```diff
@@ -332,12 +332,6 @@ HasUsers = new Mongo.Collection("hasUsers");  // dummy collection defined above
 Backers = new Mongo.Collection("backers");  // pseudo-collection defined above
 ReferralInfo = new Meteor.Collection("referralInfo"); // pseudo-collection
 
-if (Meteor.settings.public.quotaEnabled) {
-  window.testDisableQuotaClientSide = function () {
```

Comment:
Removing this makes it harder for us to test that quota is enforced server-side even if the client bypasses it.